catalog/spatial/kml/spatial-kml-transformer/src/test/java/org/codice/ddf/spatial/kml/transformer/KMLTransformerImplTest.java

```diff
@@ -247,7 +269,27 @@ public void testTransformMetacardGetsDefaultStyle()
     metacard.setLocation(POINT_WKT);
     BinaryContent content = kmlTransformer.transform(metacard, null);
     assertThat(content.getMimeTypeValue(), is(KMLTransformerImpl.KML_MIMETYPE.toString()));
-    IOUtils.toString(content.getInputStream());
+    IOUtils.toString(content.getInputStream(), StandardCharsets.UTF_8);
+  }
+
+  @Test
+  public void testTransformMetacardFromUpstreamResponse()
+      throws CatalogTransformerException, IOException, XpathException, SAXException {
+
+    MetacardImpl metacard = createMockMetacard();
+    metacard.setLocation(POINT_WKT);
+
+    Result result = mock(Result.class);
```

Comment:
✏️ I don't think you really need to mock the result; it can be instantiated with `metacard`

Reply:
Fixed.